set.mm

```diff
@@ -30829,6 +30838,15 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
       ABEZFZCGABCHZCGQIJABGRSCACBKLCQMABCACNOP $.
   $}
 
+  ${
+    $d x y A $.  $d y V $.
+    $( An element of a class exists.  (Contributed by NM, 1-May-1995.)  Reduce
+       dependencies on axioms.  (Revised by BJ, 29-Apr-2019.) $)
+    elisset $p |- ( A e. V -> E. x x = A ) $=
+      ( vy cv wceq wcel wa wex exsimpl dfclel eqeq1 cbvexvw 3imtr4i ) DEZBFZOCG
+      ZHDIPDIBCGAEZBFZAIPQDJDBCKSPADROBLMN $.
+  $}
```

Comment:
I see, this is basically bj-elisset(v), thanks. I'll take care of my mathbox. Same for bj-iiseti(v).

Reply:
Actually, if you want, you can add a "revised by SN" tag.